### PR TITLE
Replace almost all renasar references with monorail. Update http conf…

### PR DESCRIPTION
**Pull request by benbp**
…iguration. Various fixes

Replaced renasar references. I kept in renasar-ansible.pub since that's still used.

I also removed some things in the ubuntu_installation_instructions with TODO markers, hope that's ok.

Updated the default config file. Copied a working one out of the stacks but sanitized the shared key.

Replaced installation directories with placeholders, since this isn't being distributed as one of our OVA files.

### CODE REVIEW

**Comment by benbp** [comment]
@heckj @bossidy @christineday

File: docs/monorail/skus.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ be applied.
 API commands
 ^^^^^^^^^^^^^^^^^^^^^^
 
-When running the renasar-http process, these are some common API commands you
+When running the on-process, these are some common API commands you
```
Comment (heckj) [inline docs/monorail/skus.rst]: should this be `on-http process` instead of `on-process`?

File: docs/monorail/ubuntu_install_instructions.rst [hunk 2]
```diff
@@ -261,7 +219,7 @@ To create workflows, reference them by the “injectableName” property, i.e.
 
     curl -X POST localhost/api/1.1/nodes/<identifier>/workflows?name=Graph.InstallUbuntu
 
-There is now basic support for a custom type workflow that takes a list of shell commands and runs them. In order to utilize this feature, new workflows and tasks (units of work) must be registered in the system. Note that currently, these are only stored in memory, so they must be recreated whenever the renasar-taskgraph process is restarted. To create a basic workflow that runs user specified shell commands, with user specified images, the following steps are necessary:
+There is now basic support for a custom type workflow that takes a list of shell commands and runs them. In order to utilize this feature, new workflows and tasks (units of work) must be registered in the system. Note that currently, these are only stored in memory, so they must be recreated whenever the on-process is restarted. To create a basic workflow that runs user specified shell commands, with user specified images, the following steps are necessary:
```
Comment (heckj) [inline docs/monorail/ubuntu_install_instructions.rst]: `on-process` --> `on-taskgraph process`

File: docs/monorail/ubuntu_install_instructions.rst [hunk 3]
```diff
@@ -60,80 +60,39 @@ Set the Links to the Mirrors
...
-    curl http://xfer.renasar.com/emc/ansible_test.tar.gz | tar xvzf -
+  TODO: update instructions here
 
 Configure the applications to start
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-All of the renasar programs are set to respect and verify the existance of a
-default configuration file. If this file doesn't exist, the services will
+All of the monorail programs are set to respect and verify the existence of a
+default configuration file (/opt/onrack/etc/monorail.json). If this file doesn't exist, the services will
```
Comment (heckj) [inline docs/monorail/ubuntu_install_instructions.rst]: is "onrack" in our code as the place it looks? If not, can we make this "/opt/monorail/etc/monorail.json"?

Comment (benbp) [inline docs/monorail/ubuntu_install_instructions.rst]: It's hardcoded to that right now in our constants file (https://github.com/RackHD/on-core/blob/master/lib/common/constants.js#L54). I can make a ticket to change the default and notify the corsair team how to change it, would that work?

Comment (benbp) [inline docs/monorail/ubuntu_install_instructions.rst]: Also, maybe better as /opt/monorail/config.json or something?

Comment (heckj) [inline docs/monorail/ubuntu_install_instructions.rst]: Yep - better to be accurate now - we've just been purging references to "onrack", but this one's baked in a bit right now...

**Comment by benbp** [comment]
Updated on-process errors @heckj

**Comment by heckj** [comment]
+1

**Pull request merged**